### DISCUSSION TITLE
Muse Hub seems anti-user

### DISCUSSION THREAD

**Discussion by anna-s-h** [musescore/MuseScore#16129] [General]
MuseScore is an open source project which is available on Linux. Torrent client aside, MuseHub as a background process which is privileged, on by default, and enforces DRM, is strongly against the spirit of the project. Will there be documentation on developing and using high-quality soundfonts compatible with MS4 which aren't anti-user?

**Comment by cbjeukendrup** [comment]
> MuseScore as a background process

Nope, that's not MuseScore, that's Muse Hub, which in turn is not the same as Muse Sounds. If you want to complain about Muse Hub, go to https://musehub.zendesk.com/hc/en-gb, but I'd recommend to refrain from saying things that have already been said because that doesn't bring anyone any further. 

Also, it is already possible to create high-quality _SoundFonts_ for MuseScore, because SoundFont is a specific open standard for packaging samples in a file, and has no relation to Muse or MuseScore.

But I understand that this is probably not what you mean, but you want to create sounds using the same API as Muse Sounds. Currently there is no documentation yet, partly because it's still in development. Personally I don't know about the details, but it has been said that there are plans to make public documentation in the future.

**Reply by anna-s-h** [reply]
You are correct, I misspoke. I was aware of the Musehub forums but thought this seemed a better place to ask about MuseSounds alternatives, since the playback format is part of the MuseScore project. I'm glad there are plans to make such things possible. I hope we don't have to wait too long. Thanks for the response.
